README.md

```diff
@@ -1,2 +1,239 @@
+
+# Troubleshooting & miscellaneous
+
+### preCICE is not found
+
+The following error shows up during installation, if preCICE is not found:
+
+```
+  /tmp/pip-install-d_fjyo1h/pyprecice/precice.cpp:643:10: fatal error: precice/SolverInterface.hpp: No such file or directory
+    643 | #include "precice/SolverInterface.hpp"
+        |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  compilation terminated.
+  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
+  ----------------------------------------
+  ERROR: Failed building wheel for pyprecice
+Failed to build pyprecice
+ERROR: Could not build wheels for pyprecice which use PEP 517 and cannot be installed directly
+```
+
+There are two possible reasons, why preCICE is not found:
+
+1. preCICE is not installed. Please download and install the C++ library preCICE. See [above](https://github.com/precice/python-bindings/blob/develop/README.md#required-dependencies).
+2. preCICE is installed, but cannot be found. Please make sure that preCICE can be found during the installation process. See our wiki page on [linking to preCICE](https://github.com/precice/precice/wiki/Linking-to-preCICE) and [the instructions above](https://github.com/precice/python-bindings/blob/develop/README.md#precice-at-custom-location-setting-paths).
+
+### Version of Cython is too old
+
+In case the compilation fails with `shared_ptr.pxd not found` messages, check if you use the latest version of Cython.
+
+### Version of pip3 is too old
+
+If you see the following error
+```
+error: option --single-version-externally-managed not recognized
+```
+your version of pip might be too old. Please update pip and try again. One possible way for updating pip is to run the following commands:
+```
+wget -q https://bootstrap.pypa.io/get-pip.py -O get-pip.py && python3 get-pip.py
+```
+*Be aware that `python3 get-pip.py` might require root privileges.*
+
+Check your version of pip via `pip3 --version`. For version 8.1.1 and 9.0.1 we know that this problem occurs. *Remark:* you get versions 8.1.1 of pip if you use `sudo apt install python3-pip` on Ubuntu 16.04 (pip version 9.0.1 on Ubuntu 18.04)
+
+### I'm using preCICE < 2.0.0, but there is no matching version of the bindings. What can I do?
+
+If you want to use the old python bindings (released with preCICE version < 2.0.0), please refer to the documentation of the corresponding preCICE version. 
+
+### Installing the python bindings for Python 2.7.17
+
+*Note that the instructions in this section are outdated and refer to the deprecated python bindings. Until we have updated information on the installation procedure for the python bindings under this use-case, we will keep these instructions, since they might still be very useful* (Originally contributed by [huangq1234553](https://github.com/huangq1234553) to the precice wiki in [`precice/precice/wiki:8bb74b7`](https://github.com/precice/precice/wiki/Dependencies/8bb74b78a7ebc54983f4822af82fb3d638021faa).)
```

Comment:
You  could put this into:

<details><summary>Click to travel back in time.</summary>
Hello
</details>

Reply:
Done. See 3be9aaf